default.yml

```diff
@@ -1,7 +1,7 @@
 require: rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.7
```

Comment:
I think it is safe to bump this up to target Ruby 2.7 as the default since we no longer have any services depending on 2.4